server/app/rest_api/rest.py

```diff
@@ -45,7 +45,7 @@ def get(self):
                 },
                 "links": {"about-dataset": current_app.config["ABOUT_DATASET"]},
                 "parameters": {**current_app.data.get_config_parameters(uid=cxguid, collection=anno_collection)},
-                "library_versions": {"cellxgene": cellxgene_version, "anndata": anndata_version},
+                "library_versions": {"cellxgene": cellxgene_version, "anndata": str(anndata_version)},
```

Comment:
uh, this looks like a bug on our part, `__version__` should always be a string I think!